src/main/java/org/jenkinsci/plugins/codesonar/models/SearchConfigData.java

```diff
@@ -9,7 +9,7 @@
  * @author aseno
  *
  */
-public class AnalysisProceduresConfigData {
+public class SearchConfigData {
```

Comment:
Generalized the name of this class because it can be used by all those APIs that contemplate a `*_json` parameter.